react/constants/propTypes.js

```diff
@@ -133,11 +133,23 @@ export const mapType = PropTypes.string
 
 export const orderType = PropTypes.string
 
+export const schemaLayoutPropTypes = {
```

Comment:
I don't think we need to separate the schema prop types by content vs layout, you can include this directly in the existing schema prop types object